README.md

```diff
@@ -144,7 +164,7 @@ mesh profile dump --set profile=minimal
 
 #### Select a specific configuration profile
 
-The simplest customization is to select a profile different to `default` e.g. `sds`. Create the following config file:
+The simplest customization is to select a profile different to `default` e.g. `sds`. See [samples/sds.yaml](samples/sds.yaml):
 
 ```yaml
```

Comment:
Do we still need to keep the below part or just pointing to samples is enough

Reply:
I think we should keep the text if users just want to read it.